tlatools/org.lamport.tlatools/src/tla2sany/parser/Operators.java

```diff
@@ -1,121 +1,149 @@
-// Copyright (c) 2003 Compaq Corporation.  All rights reserved.
```

Comment:
I don't know all the rules, but I think these copyright notices should be retained unless the file is _completely_ deleted or replaced. (You've rewritten a lot of this file, but several names/comments/methods are unchanged, only reindented.)

Reply:
Maybe a @lemmy question, to my understanding Compaq is defunct and everything to do with copyright is assigned to the Linux Foundation?

Reply:
IANAL, but I would just leave it there.